kratos/includes/gid_gauss_point_container.h

```diff
@@ -323,45 +424,65 @@ class GidGaussPointsContainer
         if( mMeshElements.size() != 0 || mMeshConditions.size() != 0 )
         {
             //WriteGaussPoints(ResultFile);
-            GiD_fBeginResult( ResultFile, (char *)(rVariable.Name()).c_str(), (char *)("Kratos"), SolutionTag,
-                             GiD_Matrix, GiD_OnGaussPoints, mGPTitle, NULL, 0, NULL );
-
-            std::vector<Vector> ValuesOnIntPoint(mSize);
+            bool isEntityActive = false;
```

Comment:
Idem, with https://github.com/KratosMultiphysics/Kratos/pull/10311 can be simplified

Reply:
OK @loumalouomega will wait for #10311